frontend/src/metabase/entities/containers/EntityName.unit.spec.tsx

```diff
@@ -24,4 +24,23 @@ describe("EntityName", () => {
       expect(await screen.findByText("Testy Tableton")).toBeInTheDocument();
     });
   });
+
+  describe("questions", () => {
+    test("question with name (metabase#33192)", async () => {
```

Comment:
This test was run against the `EntityName` before the fix and it failed as expected. The easiest way is to checkout `frontend/src/metabase/entities/containers/EntityName.unit.spec.tsx` from 00446edd488c7a1da4cd5056c69f1db3fe580222.

Otherwise, if you want to run this against the `EntityName` from master which would still be a jsx file, you'd need to modify the import back to default import as well.